config/install/islandora_mirador.settings.yml

```diff
@@ -1 +1,4 @@
+mirador_library_use_remote: 'remote'
+mirador_library_location: 'https://roblib.github.io/mirador-integration-islandora/islandora-mirador-0.1.0.js'
```

Comment:
A possible response to @adam-vessey 's [concern](https://github.com/Islandora/islandora_mirador/pull/14#discussion_r1028263179) might be to remove this default, and instead, offer it in documentation. But I do not share his concern.

Reply:
I think we need a sensible default that we control. I'm happy to push this to the Islandora GitHub repo if people prefer. 

As discussed in the committers call on 2023-01-05, it would require a lot of work to compile an instance of the app with plugins in each deployment method.